driver/src/price_estimation/threaded_price_source.rs

```diff
@@ -1,12 +1,16 @@
 use super::{price_source::PriceSource, Token};
```

Comment:
Later, when we have an executor, we can refactor ThreadedPriceSource into something simpler that doesn't use threads.

Reply:
👍 It already looks like we got rid of some threading logic in the PR.